src/EmailAddressValidator.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+namespace ipl\Validator;
+
+use ipl\I18n\Translation;
+
+/**
+ * Validates Email address
+ */
+class EmailAddressValidator extends BaseValidator
+{
+    use Translation;
+
+    /**
+     * @var bool If MX check should be enabled
+     */
+    protected $mx = false;
+
+    /**
+     * @var bool If a deep MX check should be done
+     */
+    protected $deep = false;
+
+    /**
+     * Create a new E-mail address validator with optional options
+     *
+     * @param array $options
+     *
+     * available options
+     *
+     * 'mx'   => If MX check should be enabled, boolean
+     * 'deep' => If a deep MX check should be done, boolean
```

Comment:
The long description always comes after the short description (first line) and before any other sections.